libursa/Cargo.toml

```diff
@@ -111,7 +111,7 @@ aead = { version = "0.3", optional = true }
 aes = { version = "0.4", optional = true }
 aes-gcm = { version = "0.6.0", optional = true }
 amcl = { version = "0.2",  optional = true, default-features = false, features = ["bn254", "secp256k1"]}
-amcl_wrapper = {version = "0.3.5", features = ["bls381"], optional = true }
+amcl_wrapper = {version = "0.4.0", features = ["bls381"], optional = true }
```

Comment:
We actually can't use this because it updates to amcl_core vs amcl_milagro. amcl_core uses a GPLv3 license.

Reply:
Oh, they changed their license back to Apache 2.0. This is okay.